examples/lusid/corporate-actions-source/create-corporate-action-source.sql

```diff
@@ -0,0 +1,28 @@
+-- ===============================================================
+-- Description:
+-- 1. In this query, we run an ETL process on a corporate action 
+-- source.
+-- 2. Firstly, create a corporate action source and assign it to 
+-- a data table.
+-- 3. We then write this table to LUSID using the provider.
+-- ===============================================================
+
+-- Defining corporate action scope and code.
+
+@@corporateActionSourceScope = select 'luminesce-examples';
+@@corporateActionSourceCode = select 'uk-equity';
```

Comment:
I would maybe change this to something like 'example-corp-act-source' as 'uk-equity' is the unique code for a portfolio in these examples and don't want it to be confusing.